src/components/Footer.jsx

```diff
@@ -0,0 +1,123 @@
+import React from 'react';
+import Fetch from 'node-fetch';
+
+export default React.createClass({
+
+  componentDidMount() {
+    fetch('https://static.zooniverse.org/zoo-footer/zoo-footer.json')
+      .then((response) => {
+        return response.json();
+      })
+      .then((data) => {
+        this.setState(data);
+      });
+
+  },
+  getInitialState: () => {
+  return {};
+  },
+  render() {
+   let space = (this.state.projectsList && this.state.projectsList[0].projects.map((project, i) => <li key={i}><a href={project.url}>{project.name}</a></li>)) || 'Loading';
+
+    let naturalWorld = (this.state.projectsList && this.state.projectsList[1].projects.map((project, i) => <li key={i}><a href={project.url}>{project.name}</a></li>)) || 'Loading';
+
+    let humanities = (this.state.projectsList && this.state.projectsList[2].projects.map((project, i) => <li key={i}><a href={project.url}>{project.name}</a></li>)) || 'Loading';
+
+    let bioPhys = (this.state.projectsList && this.state.projectsList[3].projects.map((project, i) => <li key={i}><a href={project.url}>{project.name}</a></li>)) || 'Loading';
+    console.log(this.state);
+    return (
+      <div className="site-footer">
+
+    <section className="footer-zooniverse">
+        <div className="container">
+            <svg  preserveAspectRatio="xMidYMid" viewBox="0 0 59 55">
```

Comment:
Instead of having the SVG inline, we could `require` in the SVG files and use `dangerouslySetInnerHTML` - something like http://blog.daiwei.lu/load-svg-in-react/ should work?